herd/AArch64Sem.ml

```diff
@@ -840,15 +840,7 @@ module Make
 
       let lift_kvm dir updatedb mop ma an ii mphy =
         let mfault ma a =
-          insert_commit_to_fault ma (fun _ -> mk_fault a dir an ii None) ii
-          >>!
-            begin
-              let open Precision in
-              match C.precision with
-              | Fatal -> B.Exit
-              | Skip -> B.Next []
-              | Handled -> B.ReExec
-            end in
+          insert_commit_to_fault ma (fun _ -> mk_fault a dir an ii None) ii >>! B.Fault in
```

Comment:
This is where we would ideally set `ELR_EL1` but then when I try and read `ELR_EL1` to use it in the subsequent `ERET` the value is not determined until the solver is invoked. @maranget, is there a way to get an early solution for these trivial cases?